nav2_recoveries/include/nav2_recoveries/recovery.hpp

```diff
@@ -110,12 +106,14 @@ class Recovery
       node_, footprint_topic);
 
     collision_checker_ = std::make_unique<nav2_costmap_2d::CollisionChecker>(
-      *costmap_sub_, *footprint_sub_, tf_, node_->get_name(), "odom");
+      *costmap_sub_, *footprint_sub_, *tf_, node_->get_name(), "odom");
 
     vel_pub_ = node_->create_publisher<geometry_msgs::msg::Twist>("cmd_vel", 1);
+
+    onConfigure();
   }
 
-  void cleanup()
+  void cleanup() override
```

Comment:
Similar to `onConfigure` shouldn't there be an `onCleanup` to handle and de-allocate any members of the derived classes?

Reply:
Sure that makes sense

Reply:
Changes have been reflected